modules/core/src/test/scala/doobie/util/StrategySuite.scala

```diff
@@ -45,18 +41,18 @@ class StrategySuite extends munit.FunSuite {
     }
 
     override lazy val ConnectionInterpreter = new ConnectionInterpreter {
-      override val close = delay(() => Connection.close = Some(())) *> super.close
-      override val rollback = delay(() => Connection.rollback = Some(())) *> super.rollback
-      override val commit = delay(() => Connection.commit = Some(())) *> super.commit
-      override def setAutoCommit(b: Boolean) = delay(() => Connection.autoCommit = Option(b)) *> super.setAutoCommit(b)
+      override val close = delay(Connection.close = Some(())) *> super.close
```

Comment:
Still feel like we shouldn't be even using `*>` with IO and always prefering `>>`, see https://gitter.im/typelevel/cats-effect?at=6062ef06eb800601df0adf70

Reply:
They're identical up to termination. I always use the weaker operator unless I know I need the stronger one.

Reply:
Yeah, Fabio (in an answer before the message you linked to) also said to use it when recursion is involved, which isn't always the case ;)